packages/bigtable/src/table.js

```diff
@@ -659,7 +677,17 @@ Table.prototype.getRows = function(options, callback) {
  *   }
  * ];
  *
- * table.insert(entries, function(err, apiResponse) {});
+ * table.insert(entries, callback);
+ *
+ * //-
+ * // A stream mode is also provided, this is useful when making a large number
```

Comment:
I'm not sure if that's quite right, that would imply to me that I can pipe in entries, where as we just stream status objects back to the user.